mobile-app/app/screens/AppNavigator/screens/Auctions/screens/AuctionDetailScreen.tsx

```diff
@@ -106,271 +122,302 @@ export function AuctionDetailScreen(
+              )}
+            >
+              {translate("components/AuctionDetailScreen", "Auction closed")}
+            </ThemedTextV2>
+          )}
 
-interface AuctionActionSectionProps {
-  minNextBidInToken: BigNumber;
-  minNextBidInUSD: BigNumber;
-  symbol: string;
-  displaySymbol: string;
-  blocksRemaining: number;
-  onQuickBid: () => void;
-  onPlaceBid: () => void;
-}
+          <ButtonV2
+            fillType="fill"
+            label={translate("components/AuctionDetailScreen", "Bid")}
+            disabled={blocksRemaining === 0}
+            styleProps="mx-7"
+            onPress={onPlaceBid}
+            testID="auction_detail_place_bid_btn"
+          />
+        </View>
 
-const AuctionActionSection = memo(
-  (props: AuctionActionSectionProps): JSX.Element => {
-    return (
-      <ThemedView
-        light={tailwind("bg-white border-gray-200")}
-        dark={tailwind("bg-gray-900 border-gray-700")}
-        style={tailwind(
-          "absolute w-full bottom-0 flex-1 border-t px-4 pt-5 pb-10"
-        )}
-      >
-        <MinNextBidTextRow
-          displaySymbol={props.displaySymbol}
-          minNextBidInToken={props.minNextBidInToken.toFixed(8)}
-          minNextBidInUSD={getPrecisedTokenValue(props.minNextBidInUSD)}
-          labelTextStyle={tailwind("text-sm items-center")}
-          valueTextStyle={tailwind("font-semibold text-base")}
-          testID="auction_detail_min_next_bid"
-        />
-        <View
-          style={tailwind("flex flex-row mt-4 items-center justify-center")}
-        >
-          <IconButton
-            iconLabel={translate("components/QuickBid", "QUICK BID")}
-            iconSize={16}
-            style={tailwind("mr-1 w-1/2 justify-center p-2 rounded-sm")}
-            onPress={props.onQuickBid}
-            disabled={props.blocksRemaining === 0}
-            textStyle={tailwind("text-base")}
-            testID="auction_details_quick_bid_button"
+        {Platform.OS === "web" && (
+          <BottomSheetWebWithNav
+            modalRef={containerRef}
+            screenList={bottomSheetScreen}
+            isModalDisplayed={isModalDisplayed}
           />
-          <IconButton
-            iconLabel={translate("components/AuctionDetailScreen", "PLACE BID")}
-            iconSize={16}
-            style={tailwind(
-              "ml-1 w-1/2 justify-center p-2 rounded-sm bg-primary-50"
-            )}
-            onPress={props.onPlaceBid}
-            disabled={props.blocksRemaining === 0}
-            themedProps={{
-              light: tailwind("bg-primary-50 border-primary-50"),
-              dark: tailwind("bg-darkprimary-700 border-darkprimary-700"),
-            }}
-            textStyle={tailwind("text-base")}
-            textThemedProps={{
-              light: tailwind("text-primary-500"),
-              dark: tailwind("text-white"),
+        )}
+
+        {Platform.OS !== "web" && (
+          <BottomSheetWithNav
```

Comment:
we can remove bottomsheet related code because no more quick bid in detail screen